c7n/resources/emr.py

```diff
@@ -84,9 +84,13 @@ def consolidate_query_filter(self):
         return result
 
     def augment(self, resources):
+        client = local_session(self.get_resource_manager('emr').session_factory).client('emr')
         # remap for cwmetrics
         for r in resources:
             r['ClusterId'] = r['Id']
+            cluster = client.describe_cluster(ClusterId=r['ClusterId'])
+            resource_tags = cluster['Cluster']['Tags']
```

Comment:
describe emr actually has quite a bit of other useful information in it thats not in the listing, i suggest we discard the list result and replace the resource with the describe output. i need this as well for some of the emr luks checks which need securityconfiguration which isn't in the list output.